From 9e83c1dfde8d9060b61ce9f8fa5c542671bf210e Mon Sep 17 00:00:00 2001
From: Pawel Langowski <pawel.langowski@3mdeb.com>
Date: Tue, 16 Jan 2024 14:20:10 +0100
Subject: [PATCH 7/8] drivers: spidev: add spidev to compatible list

By default, spidev shows a warning if a node in the device tree's "compatible
attribute contains "spidev". This is however not always desired. This commit
adds "spidev" to the list of compatible devices in order to disable the warning.

Signed-off-by: Pawel Langowski <pawel.langowski@3mdeb.com> 
---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index be503a0e6ef7..80c5c097b78f 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -678,6 +678,7 @@ static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "lwn,bk4" },
 	{ .compatible = "dh,dhcom-board" },
 	{ .compatible = "menlo,m53cpld" },
+	{ .compatible = "spidev" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
2.25.1

